src/types.ts

```diff
@@ -1,18 +1,24 @@
+export type Length = {
+  imperial: string;
+  metric: string;
+};
+export type LinkProps = {
+  isActive: boolean;
+};
+
 export type BreedType = {
-  weight: { imperial: string; metric: string };
-  height: {
-    imperial: string;
-    metric: string;
-  };
+  weight: Length;
+  height: Length;
   id: number;
   name: string;
   bred_for: string;
   breed_group: string;
   life_span: string;
   temperament: string;
   origin: string;
-  reference_image_id: string;
-  image: {
+  reference_image_id?: string;
+  description: string;
+  image?: {
```

Comment:
make sense to move Image to separate type and use everywhere across your components (for example in SingleBreedSpecsProps types and SingleBreed component)